package.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "project-name",
   "description": "",
-  "version": "0.0.0",
+  "version": "0.3.0",
```

Comment:
All I did was matching to the current release number on github:
https://github.com/ginetta/skeleton/releases

Will take care of proper releases once this is in and the CI part is done